### ISSUE TITLE
Usar minizinha chip 3 como terminal

### ISSUE THREAD

**Issue by wesleycavalini** [pagseguro/plugpag#62]
Olá, tenho uma minizinha chip 3 e queria integrar ela com um app android.
Não achei muita documentação sobre ela, essa maquininha tem suporte ao PlugPag?

**Comment by miguelCasagrande** [comment]
Sim, a minizinha, independente de qual, é suportada. (https://dev.pagbank.uol.com.br/v1/docs/plugpag-dispositivos-suportados)

Porém, vale ressaltar que para a minizinha chip 1, pode ter algumas restrições de bluetooth, de acordo com a resposta do @hhideki no Issue #13
